src/Bootstrappers/Integrations/PostgresRLSBootstrapper.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Stancl\Tenancy\Bootstrappers\Integrations;
+
+use Illuminate\Contracts\Config\Repository;
+use Illuminate\Database\DatabaseManager;
+use Stancl\Tenancy\Contracts\TenancyBootstrapper;
+use Stancl\Tenancy\Contracts\Tenant;
+use Stancl\Tenancy\Database\Contracts\TenantWithDatabase;
+
+/**
+ * Set Postgres credentials to the tenant's credentials and use Postgres as the central connection.
+ *
+ * This bootstrapper is intended to be used with Postgres RLS (single-database tenancy).
+ *
+ * @see \Stancl\Tenancy\Commands\CreatePostgresUserForTenants
+ * @see \Stancl\Tenancy\Commands\CreateRLSPoliciesForTenantTables
+ */
+class PostgresRLSBootstrapper implements TenancyBootstrapper
+{
+    protected array $originalCentralConnectionConfig;
+    protected array $originalPostgresConfig;
+
+    public function __construct(
+        protected Repository $config,
+        protected DatabaseManager $database,
+    ) {
+        $this->originalCentralConnectionConfig = $config->get('database.connections.central');
+        $this->originalPostgresConfig = $config->get('database.connections.pgsql');
```

Comment:
This was added here since the last review, right? Is the point of it to make sure that the `central` connection will be based on the `pgsql` "preset" even if it'd be e.g. a mysql config by default?

I'm not 100% sure if that's what we want. If we do go with this approach, we should make the connection name configurable I think since `pgsql` is just what the default Laravel skeleton uses, but it's not necessarily what it's always called I think.

But more importantly I'm not sure if this is even needed/makes sense. What'd be the use case for having a central connection that's **not** postgres if we want to use Postgres RLS? The point of the feature is to in a sense add automatic scoping to a **single-database postgres-based** multi-tenant app. So I don't see a case where the central connection would not be postgres.

Reply:
This review relates to the other changes here as well, especially lines 42-45.

Reply:
> since pgsql is just what the default Laravel skeleton uses, but it's not necessarily what it's always called I think.

I think this is true. We should also probably add a static property for configuring the postgres connection name (default would be `pgsql`)

Reply:
> What'd be the use case for having a central connection that's not postgres if we want to use Postgres RLS? The point of the feature is to in a sense add automatic scoping to a single-database postgres-based multi-tenant app. So I don't see a case where the central connection would not be postgres.

I think I did this essentially to keep the config the same except the username and password.  I'm going to look into a better way of doing this, but importantly, it was because of the credentials

Reply:
Yeah, no need to touch or configure the postgres config at all. So the static property isn't needed. I made some changes so that the bootstrapper's cleaner.

https://github.com/archtechx/tenancy/pull/1105/commits/a568aa36264ee06ad0385094a5c3a57d2d8eb843

Reply:
Updated this so that the pgsql connection isn't touched
https://github.com/archtechx/tenancy/pull/1105/commits/a568aa36264ee06ad0385094a5c3a57d2d8eb843